src/Event/CfpEndingSoonEvent.php

```diff
@@ -41,25 +41,26 @@ public function buildAttachment(): array
         $cfpAttachment = SlackNotifier::ATTACHMENT;
         $template = '🔊  CFP for %s (%s) is closing %s';
         $conferenceLink = sprintf('<%s|%s>', $this->conference->getCfpUrl(), $this->conference->getName());
+        $countdown = 'in *' . $this->remainingDays . ' day' . $this->remainingDays > 1 ? 's' : '' . '* !';
```

Comment:
et les pluriels hein ? :stuck_out_tongue:

Reply:
Yes, it's mentioned in #47